tests/test_forms/test_forms.py

```diff
@@ -0,0 +1,68 @@
+from django.test import TestCase
+
+from taxi.forms import CarForm, DriverLicenseUpdateForm
+from taxi.models import Driver, Manufacturer
+
+
+class TestForm(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="Fasd",
+            country="asd"
+        )
+        self.driver1 = Driver.objects.create_user(
+            username="Driver1",
+            password="1234",
+            license_number="TES12345"
+        )
+        self.driver2 = Driver.objects.create_user(
+            username="Driver2",
+            password="1234",
+            license_number="TES12341"
+        )
+
+    def test_valid_data_car(self):
+        form_data = {
+            "model": "Test10",
+            "manufacturer": 1,
+            "drivers": [1, 2]
```

Comment:
The `drivers` field in `form_data` is set to `[1, 2]`, which assumes the IDs of the drivers. Instead, use `[self.driver1.id, self.driver2.id]` to ensure the correct IDs are used.